### ISSUE TITLE
vscode not opening

### ISSUE THREAD

**Issue by 5nowx** [skycocker/chromebrew#2886]
## Description
Installed the 'code' package and all it's dependencies, and nothing opens

If you executed a specific command:
```shell
crew update & crew upgrade
crew install code
code
```

Output:
```shell
(nothing)
```

## Versions and system information
chromebrew: `1.0.3`

ChromeOS: `CHROMEOS_AUSERVER=https://tools.google.com/service/update2
CHROMEOS_BOARD_APPID={F26D159B-52A3-491A-AE25-B23670A66B32}
CHROMEOS_CANARY_APPID={90F229CE-83E2-4FAF-8479-E368A34938B1}
CHROMEOS_DEVSERVER=
CHROMEOS_RELEASE_APPID={F26D159B-52A3-491A-AE25-B23670A66B32}
CHROMEOS_RELEASE_BOARD=link-signed-mp-v5keys
CHROMEOS_RELEASE_BRANCH_NUMBER=78
CHROMEOS_RELEASE_BUILDER_PATH=link-release/R69-10895.78.0
CHROMEOS_RELEASE_BUILD_NUMBER=10895
CHROMEOS_RELEASE_BUILD_TYPE=Official Build
CHROMEOS_RELEASE_CHROME_MILESTONE=69
CHROMEOS_RELEASE_DESCRIPTION=10895.78.0 (Official Build) stable-channel link 
CHROMEOS_RELEASE_KEYSET=mp-v5
CHROMEOS_RELEASE_NAME=Chrome OS
CHROMEOS_RELEASE_PATCH_NUMBER=0
CHROMEOS_RELEASE_TRACK=stable-channel
CHROMEOS_RELEASE_VERSION=10895.78.0
DEVICETYPE=CHROMEBOOK
GOOGLE_RELEASE=10895.78.0`

Model: Chromebook Pixel 2013 - Codename 'Link'

Architecture: `x86_64`

## Addtional information
Mention things we might need to know. Like:

What I think needs to be done:
- [ ] install code
- [ ] try to run it, nothing happens

---

**Comment by JL2210** [comment]
Restart your Chromebook, run `code`, and wait. It will work, eventually.

**Comment by uberhacker** [comment]
@5nowx: If this is the first time you have installed a gui app, there are a ton of dependencies needed for code and it can take a LONG time to resolve.  Be patient and you will be prompted for the dependencies.  It may take a few minutes.

**Comment by uberhacker** [comment]
Oh, I thought he was trying to install the vscode package initially.  If that is the case, then that package will likely need to be uninstalled first.

**Comment by 5nowx** [comment]
@uberhacker  Yes, installed all the dependencies, also this is the first GUI app i run.

I opened it 5 mins ago and still nothing, but i'll wait, if i do a "ps aux | grep 'code' " i can see the process, in an hour i'll let you know if something happens.

**Comment by 5nowx** [comment]
Yeaa... didnt opened 👎

**Comment by etbusch** [comment]
Is the sommelier daemon running? try running `startsommelier` before `code`

**Comment by 5nowx** [comment]
i just started sommelier:

```shell
chronos@localhost / $ startsommelier 
_XSERVTransmkdir: ERROR: euid != 0,directory /tmp/.X11-unix will not be created.
Warning: Unknown scale factor: 1667
Warning: Unknown scale factor: 1429
Warning: Unknown scale factor: 1111
Warning: Unknown scale factor: 909
Warning: Unknown scale factor: 667
sommelier process 13670 is running
```

i'll let it run and come back to you in 1/2 hour if opened or didn't

**Comment by etbusch** [comment]
It should open immediately. Kill any running code processes and re-run the command now that sommelierd is running.

**Comment by 5nowx** [comment]
I tried that and rerunned the command but nothing opened 👎

**Comment by uberhacker** [comment]
@5nowx: Based on the error message, it looks like you missed the post install message from sommelier.  Try `crew remove sommelier && crew install sommelier` and follow the post install instructions in blue text.

**Comment by skiman6010** [comment]
I CAN HELP WITH THIS! 

`crew remove eudev && crew install eudev`

**Comment by 5nowx** [comment]
thanks @uberhacker this worked right away, during the installation of all of the dependencies, the post-install must have been lost in the textwall of the output
Just to summerize: 

1. crew remove sommelier && crew install sommelier
2. Executed the blue text line by line
3. Restarted the machine
4. Run code

**Comment by nosarthur** [comment]
I have sommelier running and the blue post installation stuff in the .bashrc, but `code` still does not run. 

I have tried to reinstall sommelier, code, and reboot. Is there something else I can try?

My Samsung Chromebook Pro and the OS is Version 72.0.3609.3 (Official Build) dev (64-bit)

**Comment by uberhacker** [comment]
`stopsommelier && startsommelier`?  Do you get any output from `echo $DISPLAY` or `echo $SCALE`?

**Comment by nosarthur** [comment]
```
chronos / $ echo $DISPLAY
:0
chronos / $ echo $SCALE
1
```
Sometimes running `stopsommelier && startsommelier` gives rise to this message

```
chronos / $ stopsommelier && startsommelier 
Password: 
(EE) failed to read Wayland events: Broken pipe
/usr/local/sbin/sommelierd: line 2:  5192 Terminated              sommelier -X --x-display=$DISPLAY --scale=$SCALE --no-exit-with-child /bin/sh -c "/usr/local/etc/sommelierrc"
sommelier process stopped
Warning: Unknown scale factor: 1667
Warning: Unknown scale factor: 1429
Warning: Unknown scale factor: 1111
Warning: Unknown scale factor: 909
Warning: Unknown scale factor: 667
sommelier process 8091 is running
```

**Comment by JL2210** [comment]
@nosarthur The errors are normal. You can completely ignore them. If anything, restarting may help at this point.

**Comment by uberhacker** [comment]
You might try `crew remove eudev && crew install eudev`.  I know this fixed the issue for @skiman6010 in the past.

**Comment by JL2210** [comment]
Yeah.

**Comment by nosarthur** [comment]
It works now! Thanks a lot guys!
